### PR TITLE
[Localization] New translations batch locales (Move-disabling effects, abilities triggers, error messages and some misc ajustements)

### PR DESCRIPTION
**Pull request by Adri1**
## What are the changes the user will see?
Added missing translations to locales (Move-disabling effects, abilities triggers, error messages)
**I added English placeholders to facilitate suggestions, but if a language never provide any translation, I'll remove these placeholders before merge.**

## Why am I making these changes?
Having them translated

## Checklist
- [X] **I'm using `beta` as my base branch**
- [X] There is no overlap with another PR?
- [X] The PR is self-contained and cannot be split into smaller PRs?
- [X] Have I provided a clear explanation of the changes?

### CODE REVIEW

File: src/locales/zh_CN/battle.json [hunk 1]
```diff
@@ -44,6 +44,7 @@
   "moveNotImplemented": "{{moveName}}尚未实装，无法选择。",
   "moveNoPP": "这个技能的PP用完了",
   "moveDisabled": "{{moveName}}被禁用！",
+  "disableInterruptedMove": "{{pokemonNameWithAffix}}'s {{moveName}}\nis disabled!",
```
Comment (40chyan) [inline src/locales/zh_CN/battle.json:47]:
```suggestion
  "disableInterruptedMove": "{{pokemonNameWithAffix}}的{{moveName}}\n被无效化了！",
```

File: src/locales/zh_TW/battle.json [hunk 2]
```diff
@@ -40,6 +40,7 @@
   "moveNotImplemented": "{{moveName}} 未實裝，無法選擇。",
   "moveNoPP": "這個技能的PP用完了",
   "moveDisabled": "{{moveName}} 被禁用！",
+  "disableInterruptedMove": "{{pokemonNameWithAffix}}'s {{moveName}}\nis disabled!",
```
Comment (40chyan) [inline src/locales/zh_TW/battle.json:43]:
```suggestion
  "disableInterruptedMove": "{{pokemonNameWithAffix}}的{{moveName}}\n被無效化了！",
```

**Review by 40chyan — COMMENTED** [review]
zh

File: src/locales/ko/modifier-type.json [hunk 3]
```diff
@@ -47,10 +47,14 @@
       "description": "포켓몬의 성격을 {{natureName}}[[로]] 바꾸고 스타팅에도 등록한다."
     },
     "DoubleBattleChanceBoosterModifierType": {
-      "description": "{{battleCount}}번의 배틀 동안 더블 배틀이 등장할 확률이 두 배가 된다."
+      "description": "Quadruples the chance of an encounter being a double battle for up to {{battleCount}} battles."
```
Comment (EnochG1) [inline src/locales/ko/modifier-type.json:50]:
```suggestion
      "description": "{{battleCount}}번의 배틀 동안 더블 배틀이 등장할 확률이 4배가 된다."
```

File: src/locales/ko/modifier-type.json [hunk 4]
```diff
@@ -47,10 +47,14 @@
...
     },
     "TempStatStageBoosterModifierType": {
-      "description": "자신의 모든 포켓몬이 5번의 배틀 동안 {{stat}}[[가]] 한 단계 증가한다."
+      "description": "Increases the {{stat}} of all party members by {{amount}} for up to 5 battles.",
+      "extra": {
+        "stage": "1 stage",
+        "percentage": "30%"
+      }
```
Comment (EnochG1) [inline src/locales/ko/modifier-type.json:57]:
```suggestion
      "description": "자신의 모든 포켓몬이 {{amount}}번의 배틀 동안 {{stat}}[[가]] 한 단계 증가한다.",
      "extra": {
        "stage": "1 스테이지",
        "percentage": "30%"
      }
```

Comment (EnochG1) [inline src/locales/ko/modifier-type.json:57]:
```suggestion
      "description": "자신의 모든 포켓몬이 5번의 배틀 동안 {{stat}}[[가]] {{amount}}단계 증가한다.",
      "extra": {
        "stage": "1 스테이지",
        "percentage": "30%"
      }
```

File: src/locales/ja/modifier-type.json [hunk 5]
```diff
@@ -47,10 +47,14 @@
       "description": "ポケモンのせいかくを {{natureName}}にかえて スターターのせいかくをえいきゅうにかいじょする"
     },
     "DoubleBattleChanceBoosterModifierType": {
-      "description": "バトル{{battleCount}}かいのあいだ ダブルバトルになるかくりつを2ばいにする"
+      "description": "バトル{{battleCount}}かいのあいだ ダブルバトルになるかくりつを4ばいにする"
```
Comment (Chapybara-jp) [inline src/locales/ja/modifier-type.json:50]:
```suggestion
      "description": "バトル{{battleCount}}回の間 　ダブルバトルになる　　確率を　4倍に　する"
```

File: src/locales/ja/modifier-type.json [hunk 6]
```diff
@@ -47,10 +47,14 @@
...
     },
     "TempStatStageBoosterModifierType": {
-      "description": "すべてのパーティメンバーの {{stat}}を5かいのバトルのあいだ 1だんかいあげる"
+      "description": "Increases the {{stat}} of all party members by {{amount}} for up to 5 battles.",
```
Comment (Chapybara-jp) [inline src/locales/ja/modifier-type.json:53]:
```suggestion
      "description": "全員の　手持ちポケモンの　{{stat}}を　最大５回の　バトルの間に　{{amount}}あげる.",
```

File: src/locales/ja/modifier-type.json [hunk 7]
```diff
@@ -47,10 +47,14 @@
...
+      "extra": {
+        "stage": "1 stage",
+        "percentage": "30%"
```
Comment (Chapybara-jp) [inline src/locales/ja/modifier-type.json:56]:
```suggestion
        "stage": "１段階",
        "percentage": "３０％"
```

**Review by Chapybara-jp — CHANGES_REQUESTED** [review]

**Review by Chapybara-jp — DISMISSED** [review]

File: src/locales/de/modifier-type.json [hunk 8]
```diff
@@ -47,10 +47,14 @@
       "description": "Ändert das Wesen zu {{natureName}}. Schaltet dieses Wesen permanent für diesen Starter frei."
     },
     "DoubleBattleChanceBoosterModifierType": {
-      "description": "Verdoppelt die Wahrscheinlichkeit, dass die nächsten {{battleCount}} Begegnungen mit wilden Pokémon ein Doppelkampf sind."
+      "description": "Quadruples the chance of an encounter being a double battle for up to {{battleCount}} battles."
```
Comment (CodeTappert) [inline src/locales/de/modifier-type.json:50]:
```suggestion
      "description": "Vervierfacht die Chance, dass ein Kampf ein Doppelkampf wird, für bis zu {{battleCount}} Kämpfe."
```

File: src/locales/de/modifier-type.json [hunk 9]
```diff
@@ -47,10 +47,14 @@
...
     },
     "TempStatStageBoosterModifierType": {
-      "description": "Erhöht die {{stat}} aller Teammitglieder für 5 Kämpfe um eine Stufe."
+      "description": "Increases the {{stat}} of all party members by {{amount}} for up to 5 battles.",
+      "extra": {
+        "stage": "1 stage",
+        "percentage": "30%"
+      }
```
Comment (CodeTappert) [inline src/locales/de/modifier-type.json:57]:
```suggestion
  "description": "Erhöht {{stat}} aller Teammitglieder um {{amount}} für bis zu 5 Kämpfe.",
  "extra": {
    "stage": "eine Stufe",
    "percentage": "30%"
  }
}
```

File: src/locales/it/menu.json [hunk 10]
```diff
@@ -51,5 +51,7 @@
   "renamePokemon": "Rinomina un Pokémon",
   "rename": "Rinomina",
   "nickname": "Nickname",
-  "errorServerDown": "Poffarbacco! C'è stato un errore nella comunicazione col server.\n\nPuoi lasciare questa finestra aperta,\nil gioco si riconnetterà automaticamente."
-}
\ No newline at end of file
+  "errorServerDown": "Poffarbacco! C'è stato un errore nella comunicazione col server.\n\nPuoi lasciare questa finestra aperta,\nil gioco si riconnetterà automaticamente.",
+  "noSaves": "You don't have any save files on record!",
+  "tooManySaves": "You have too many save files on record!"
```
Comment (NicusPulcis) [inline src/locales/it/menu.json:56]:
```suggestion
  "noSaves": "Non ci sono file di salvataggio registrati!",
  "tooManySaves": "Ci sono troppi file di salvataggio registrati!"
```

**Review by NicusPulcis — DISMISSED** [review]
italian thingies approved and localization for the new two texts added

File: src/locales/zh_CN/modifier-type.json [hunk 11]
```diff
@@ -47,10 +47,14 @@
       "description": "将一只宝可梦的性格改为{{natureName}}并为\n该宝可梦永久解锁该性格。"
     },
     "DoubleBattleChanceBoosterModifierType": {
-      "description": "接下来的{{battleCount}}场战斗是双打的概率翻倍。"
+      "description": "Quadruples the chance of an encounter being a double battle for up to {{battleCount}} battles."
```
Comment (40chyan) [inline src/locales/zh_CN/modifier-type.json:50]:
```suggestion
      "description": "遭遇双打概率提升四倍，持续{{battleCount}}场战斗。"
```

File: src/locales/zh_CN/modifier-type.json [hunk 12]
```diff
@@ -47,10 +47,14 @@
...
     },
     "TempStatStageBoosterModifierType": {
-      "description": "为所有成员宝可梦提升一级{{stat}}，持续5场战斗。"
+      "description": "Increases the {{stat}} of all party members by {{amount}} for up to 5 battles.",
```
Comment (40chyan) [inline src/locales/zh_CN/modifier-type.json:53]:
```suggestion
      "description": "提升全队的{{stat}}{{amount}}级，持续5场战斗。",
```

File: src/locales/zh_CN/modifier-type.json [hunk 13]
```diff
@@ -47,10 +47,14 @@
...
+      "extra": {
+        "stage": "1 stage",
```
Comment (40chyan) [inline src/locales/zh_CN/modifier-type.json:55]:
```suggestion
        "stage": "1阶",
```

File: src/locales/zh_CN/menu.json [hunk 14]
```diff
@@ -51,5 +51,7 @@
   "renamePokemon": "给宝可梦起名",
   "rename": "起名",
   "nickname": "昵称",
-  "errorServerDown": "糟糕！访问服务器时发生了错误。\n\n你可以保持页面开启，\n游戏会自动重新连接。"
-}
\ No newline at end of file
+  "errorServerDown": "糟糕！访问服务器时发生了错误。\n\n你可以保持页面开启，\n游戏会自动重新连接。",
+  "noSaves": "You don't have any save files on record!",
+  "tooManySaves": "You have too many save files on record!"
```
Comment (40chyan) [inline src/locales/zh_CN/menu.json:56]:
```suggestion
  "noSaves": "你没有任何记录文件！",
  "tooManySaves": "你的记录文件太多了！"
```

File: src/locales/zh_TW/modifier-type.json [hunk 15]
```diff
@@ -47,10 +47,14 @@
       "description": "將一隻寶可夢的性格改爲{{natureName}}併爲該寶可\n夢永久解鎖該性格。"
     },
     "DoubleBattleChanceBoosterModifierType": {
-      "description": "接下來的{{battleCount}}場戰鬥是雙打的概率翻倍。"
+      "description": "Quadruples the chance of an encounter being a double battle for up to {{battleCount}} battles."
```
Comment (40chyan) [inline src/locales/zh_TW/modifier-type.json:50]:
```suggestion
      "description": "遭遇雙打機率提升四倍，持續{{battleCount}}場戰鬥。"
```

File: src/locales/zh_TW/modifier-type.json [hunk 16]
```diff
@@ -47,10 +47,14 @@
...
     },
     "TempStatStageBoosterModifierType": {
-      "description": "爲所有成員寶可夢提升一級{{stat}}，持續5場戰鬥。"
+      "description": "Increases the {{stat}} of all party members by {{amount}} for up to 5 battles.",
```
Comment (40chyan) [inline src/locales/zh_TW/modifier-type.json:53]:
```suggestion
      "description": "提升全隊的{{stat}}{{amount}}級，持續5場戰鬥。",
```

File: src/locales/zh_TW/modifier-type.json [hunk 17]
```diff
@@ -47,10 +47,14 @@
...
+      "extra": {
+        "stage": "1 stage",
```
Comment (40chyan) [inline src/locales/zh_TW/modifier-type.json:55]:
```suggestion
        "stage": "1階",
```

File: src/locales/zh_TW/menu.json [hunk 18]
```diff
@@ -40,5 +40,7 @@
   "loading": "加載中…",
   "playersOnline": "在線玩家",
   "yes": "是",
-  "no": "否"
-}
\ No newline at end of file
+  "no": "否",
+  "noSaves": "You don't have any save files on record!",
```
Comment (40chyan) [inline src/locales/zh_TW/menu.json:44]:
```suggestion
  "noSaves": "你沒有任何記錄檔！",
```

File: src/locales/zh_TW/menu.json [hunk 19]
```diff
@@ -40,5 +40,7 @@
...
+  "tooManySaves": "You have too many save files on record!"
```
Comment (40chyan) [inline src/locales/zh_TW/menu.json:45]:
```suggestion
  "tooManySaves": "你的記錄檔太多了！"
```

**Review by 40chyan — COMMENTED** [review]
zh

**Review by CodeTappert — APPROVED** [review]

**Review by Chapybara-jp — APPROVED** [review]

**Review by frutescens — APPROVED** [review]

**Review by EnochG1 — APPROVED** [review]

**Comment by frutescens** [comment]
Hi, has this bug been addressed in this PR? https://discord.com/channels/1125469663833370665/1282109467650953236/1282109467650953236

**Comment by Adri1** [comment]
> Hi, has this bug been addressed in this PR? https://discord.com/channels/1125469663833370665/1282109467650953236/1282109467650953236

Nope, just learned about it

**Pull request merged**